core/src/utils/helpers.ts

```diff
@@ -9,7 +9,7 @@ export const componentOnReady = (el: any, callback: any) => {
   if (el.componentOnReady) {
     el.componentOnReady().then(callback);
```

Comment:
Do you want to update this code too for consistency with below ?

```js
el.componentOnReady().then(() => callback(el));
```

Reply:
`el.componentOnReady()` returns `el` already, so we don't need to pass it in ourselves.

Reply:
Sure.
However:
- I think it could make the code easier to understand,
- I generally prefer being explicit and list the parameters. So maybe `.then((el) => callback(el))`. Being implicit makes it easy to shot yourself in the foot sometimes. Might be personal preferences.

Reply:
Yep fair point - I updated the code.